SPECS/python-rpm-generators/python-rpm-generators.spec

```diff
@@ -26,11 +31,11 @@ BuildArch:      noarch
 
 %package -n python3-rpm-generators
 Summary:        %{summary}
-Requires:       python3-setuptools
-# The point of split
-Conflicts:      rpm-build < 4.13.0.1-2
-# Breaking change, change a way how depgen is enabled
-Conflicts:      python-rpm-macros < 3-35
+Requires:       python3-packaging
+# We have parametric macro generators, we need RPM 4.16 (4.15.90+ is 4.16 alpha)
+Requires:       rpm > 4.15.90-0
```

Comment:
Not needed.
```suggestion
```

Reply:
It still needs RPM, right? just not the `> 4.15.90-0`?

Reply:
Sorry, right.:)